# Sales Data Calculator 

## Objective

To analyze a collection of sales values using Python and calculate the total sales, average sales, highest sale, and lowest sale.

In [1]:
# Sales data

sales = [
    12500, 18000, 9500, 22000, 15750,
    19800, 27500, 14300, 16500, 31000,
    11200, 24500, 17800, 29000, 13500,
    20500, 32000, 15200, 26000, 19000
]

sales

[12500,
 18000,
 9500,
 22000,
 15750,
 19800,
 27500,
 14300,
 16500,
 31000,
 11200,
 24500,
 17800,
 29000,
 13500,
 20500,
 32000,
 15200,
 26000,
 19000]

In [2]:
total_sales = sum(sales)
total_sales

395550

In [8]:
average_sales = total_sales / len(sales)
average_sales

19777.5

In [9]:
highest_sale = max(sales)
highest_sale

32000

In [10]:
lowest_sale = min(sales)
lowest_sale

9500

In [14]:
print("---------------------------")
print("      Sales Analysis       ")
print("---------------------------")
print("Total Sales   :", total_sales)
print("Average Sales :", average_sales)
print("Highest Sale  :", highest_sale)
print("Lowest Sale   :", lowest_sale)

---------------------------
      Sales Analysis       
---------------------------
Total Sales   : 395550
Average Sales : 19777.5
Highest Sale  : 32000
Lowest Sale   : 9500


## Explanation of Results

The total sales amount is ₹3,95,550 for 20 sales values.

The average sale is ₹19,777.5, which represents the average value of the sales.

The highest sale is ₹32,000, while the lowest sale is ₹9,500.

These results provide a basic overview of the sales data and help understand the overall sales performance.